File: sample-app-java/app/src/main/java/com/bluejeans/android/sdksample/SampleApplication.java
/*
 * Copyright (c) 2020 Blue Jeans Network, Inc. All rights reserved.
 */
package com.bluejeans.android.sdksample;

import android.app.Application;
import android.util.Log;
import com.bluejeans.bluejeanssdk.BlueJeansSDK;
import com.bluejeans.bluejeanssdk.BlueJeansSDKInitParams;

public class SampleApplication extends Application {

    private static BlueJeansSDK blueJeansSDK;
    private static final String TAG = "SampleApplication";

    @Override
    public void onCreate() {
        super.onCreate();
        initSDK();
        Log.i(
                TAG, "App VersionName " + BuildConfig.VERSION_NAME +
                        " App VersionCode " + BuildConfig.VERSION_CODE +
                        " SDK VersionName " + blueJeansSDK.getVersion());

    }

    private void initSDK() {
        try {
            blueJeansSDK = new BlueJeansSDK(new BlueJeansSDKInitParams(this), null);
        } catch (Exception ex) {
            Log.e(TAG, "Exception while initiating sdk " + ex.getMessage());
        }
    }

    public static BlueJeansSDK getBlueJeansSDK() {
        return blueJeansSDK;
    }
}

File: sample-app-kotlin/app/src/main/java/com/bluejeans/android/sdksample/SampleApplication.kt
/*
 * Copyright (c) 2021 Blue Jeans Network, Inc. All rights reserved.
 */
package com.bluejeans.android.sdksample

import android.app.Application
import com.bluejeans.bluejeanssdk.BlueJeansSDK
import com.bluejeans.bluejeanssdk.BlueJeansSDKInitParams

class SampleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        initSDK()
    }

    private fun initSDK() {
        try {
            blueJeansSDK = BlueJeansSDK(BlueJeansSDKInitParams(this))
        } catch (ex: Exception) {
            throw ex
        }
    }

    companion object {
        const val TAG = "SampleApplication"
        lateinit var blueJeansSDK: BlueJeansSDK
    }
}